### ISSUE TITLE
Implement `push` subcommand

### ISSUE THREAD

**Issue by abourget** [eoscanada/eosc#13]
* [ ] action
* [ ] code
* [ ] code-abi
* [x] abi
* [ ] transaction

**Comment by tampajohn** [comment]
Is code-abi going to be a port of `cleos set contract`, or am I missing something?  I personally don't like the separation in cleos of the push/set commands... so I'm assuming you want to collapse these subcommands to some degree?

**Comment by abourget** [comment]
So actually, `eosc system setcontract` is `setcode` + `setabi` .. and I've listed something I think is going to be better.  It's in `DESIGN.md` also, up for discussion.. I kinda like it. Here's a summary:

```
eosc tx create [[contract] [action] payload] -p permission@active
eosc tx create [path/to/actions.yaml] -p permission@active
eosc tx sign [transaction.yaml|json]
eosc tx push [transaction.yaml] [[transaction2.json] [[transaction3.yaml] [...]]]
eosc tx pack
eosc tx unpack
eosc tx pack-actions
eosc tx unpack-actions
eosc tx review
```
I'd like to have `.yaml` files be the base format .. since it supports comments, easier copy & paste.

Right now, `--output-transaction` outputs a JSON with the transaction at the first level. I think it'd be better to put that under something like:

```
abis:
  contract1: PACKEDABI
  contract2: PACKEDABI
transactions:
- [trx here]
- [trx here]
chain_id: 5fff1....
requested_keys:
- EOSK12345...
- EOSK1abcde...
```

This way, we'd have all we need for offline transaction signing.. _as well as_ offline transaction contents verification. Without the ABIs, you can't unpack the transaction and look what it *really* means before signing.. I'd like to nail that flow, and make it rock solid.

**Comment by abourget** [comment]
@fproulx-eoscanada ^^

**Comment by abourget** [comment]
@billettc ^^

**Comment by maoueh** [comment]
`eosc tx create` => #19

**Comment by fproulx-eoscanada** [comment]
Shipped in v1.0.0
